src/components/Header/styles.js

```diff
@@ -4,6 +4,7 @@ import { withStyles } from '@material-ui/core';
 const styles = {
   root: {
     backgroundColor: 'white',
+    boxShadow: '0 1px 10px rgba(0, 0, 0, 0.2)',
```

Comment:
Shouldn't the component have the shadow by default as per https://material-ui.com/demos/app-bar/? If you can't get the shadow to show, then just confirm that the above is per Material Design specs?

Reply:
I'm looking into this now. Seems that it should have it by default, so will look into why this is not displaying.